tests/zkevm/test_worst_compute.py

```diff
@@ -144,24 +238,212 @@ def test_worst_modexp(
+                "2BB8324AF6CFC93537A2AD1A445CFD0CA2A71ACD7AC41FADBF933C2A51BE344D",
+                "120A2A4CF30C1BF9845F20C6FE39E07EA2CCE61F0C9BB048165FE5E4DE877550",
+                # Second pairing
+                "111E129F1CF1097710D41C4AC70FCDFA5BA2023C6FF1CBEAC322DE49D1B6DF7C",
+                "103188585E2364128FE25C70558F1560F4F9350BAF3959E603CC91486E110936",
+                "198E9393920D483A7260BFB731FB5D25F1AA493335A9E71297E485B7AEF312C2",
+                "1800DEEF121F1E76426A00665E5C4479674322D4F75EDADD46DEBD5CD992F6ED",
+                "090689D0585FF075EC9E99AD690C3395BC4B313370B38EF355ACDADCD122975B",
+                "12C85EA5DB8C6DEB4AAB71808DCB408FE3D1E7690C43D37B4CE6CC0166FA7DAA",
+            ],
+            32 * 2 * 6,
+            id="bn128_pairing",
+        ),
+        pytest.param(
+            0x09,
+            [
+                "0000FFFF",
+                "48C9BDF267E6096A3BA7CA8485AE67BB2BF894FE72F36E3CF1361D5F3AF54FA5D182E6AD7F520E511F6C3E2B8C68059B6BBD41FBABD9831F79217E1319CDE05B",
+                "61626300000000000000000000000000000000000000000000000000",
+                "00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
+                "000000000300000000000000000000",
+                "00000000010000000000000000000000",
+                "00",
+            ],
+            213,
+            id="blake2f",
+        ),
+        pytest.param(
+            0x0A,
+            [
+                "01E798154708FE7789429634053CBF9F99B619F9F084048927333FCE637F549B",
+                "564C0A11A0F704F4FC3E8ACFE0F8245F0AD1347B378FBF96E206DA11A5D36306",
+                "24D25032E67A7E6A4910DF5834B8FE70E6BCFEEAC0352434196BDF4B2485D5A1",
+                "8F59A8D2A1A625A17F3FEA0FE5EB8C896DB3764F3185481BC22F91B4AAFFCCA25F26936857BC3A7C2539EA8EC3A952B7",
+                "873033E038326E87ED3E1276FD140253FA08E9FC25FB2D9A98527FC22A2C9612FBEAFDAD446CBC7BCDBDCD780AF2C16A",
+            ],
+            192,
+            id="point_evaluation",
+        ),
+        pytest.param(
+            0x0B,
+            [
+                "000000000000000000000000000000000572CBEA904D67468808C8EB50A9450C9721DB309128012543902D0AC358A62AE28F75BB8F1C7C42C39A8C5529BF0F4E00000000000000000000000000000000166A9D8CABC673A322FDA673779D8E3822BA3ECB8670E461F73BB9021D5FD76A",
+                "4C56D9D4CD16BD1BBA86881979749D280000000000000000000000000000000009ECE308F9D1F0131765212DECA99697B112D61F9BE9A5F1F3780A51335B3FF981747A0B2CA2179B96D2C0C9024E522400000000000000000000000000000000032B80D3A6F5B09F8A84623389C5F80C",
+            ],
+            256,
+            id="bls12_g1add",
+        ),
+        pytest.param(
+            0x0C,
```

Comment:
The worst input is probably the one for 128 pairs. Please try to input imputs from the existing tests instead of using raw hex values here.

Reply:
By "existing tests" you mean here https://eips.ethereum.org/assets/eip-2537/test-vectors ? There the inputs are in raw hex.

Reply:
For BLS precompiles IDs you can also use these constants: https://github.com/ethereum/execution-spec-tests/blob/main/tests/prague/eip2537_bls_12_381_precompiles/spec.py#L112-L119

Reply:
Oh, amazing. Thanks for the pointers, I'll do.